### PR TITLE
Fix/png admin v2 + align all image fields to accept the same image types

### PR DESCRIPTION
**Pull request by margulies**
In `admin_v2`, png files aren't accepted for upload. This seems to fix the issue.

### CODE REVIEW

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Comment by 0xdevalias** [comment]
https://github.com/sparkletown/sparkle/pull/821 added this to `src/components/molecules/ImageInput/ImageInput.tsx` already, but it seems we have duplicate `ImageInput` components.

---

`src/components/molecules/ImageInput/ImageInput.tsx` is used by:

- `src/pages/Admin/Venue/DetailsForm.tsx`
- `src/pages/Admin/Venue/Rooms/RoomsForm.tsx`

`src/components/atoms/ImageInput/ImageInput.tsx` is used by:

- `src/pages/Admin/BasicInfo/BasicInfo.tsx`
- `src/pages/Admin/Details/Form/DetailsForm.tsx`
- `src/pages/Admin/Room/Edit/RoomEdit.tsx`
- `src/pages/Admin/Room/Modal/Item/Item.tsx`

From a basic skim of the code, looks less than trivial to unpick them and figure the better of the two, definitely OOS of this PR.

---

It looks like there is a `ACCEPTED_IMAGE_TYPES` in `src/settings.ts` though:

- https://github.com/sparkletown/sparkle/blob/staging/src/settings.ts#L126-L127

Which is currently used in:

- `src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx`

We should also use this in both of:

- `src/components/molecules/ImageInput/ImageInput.tsx`
- `src/components/atoms/ImageInput/ImageInput.tsx`

So that the one constant controls all of them.

We can technically also import from the `src/` folder in our scripts, so we could use this constant there as well for:

- `scripts/resize-images.ts`

**Review by 0xdevalias — CHANGES_REQUESTED** [review]
As per my comments in https://github.com/sparkletown/sparkle/pull/1183#issuecomment-781826235 can we refactor all of the locations to use `ACCEPTED_IMAGE_TYPES` from `serc/settings.ts` please.

File: src/components/molecules/ImageInput/ImageInput.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { useMemo } from "react";
 import { FieldError } from "react-hook-form";
 import "firebase/functions";
-
+import { ACCEPTED_IMAGE_TYPES } from "settings";
```
Comment (0xdevalias) [inline src/components/molecules/ImageInput/ImageInput.tsx:4]:
```suggestion
import { ACCEPTED_IMAGE_TYPES } from "settings";

```

**Review by 0xdevalias — APPROVED** [review]
Looks good, LGTM!

**Comment by mobilevinay** [comment]
is this really ready for QA in staging?

**Pull request merged**